cluster/contrail-api.manifest

```diff
@@ -7,7 +7,7 @@
 	"containers":[{
 	    "name": "contrail-api",
 	    "image": "opencontrail/config:2.20",
-	    "command": ["/bin/sh", "-c", "/usr/bin/contrail-api --conf_file=/etc/contrail/contrail-api.conf 1>>/var/log/contrail/contrail-api.stdout 2>>/var/log/contrail/contrail-api.err"],
+	    "command": ["/bin/sh", "-c", "/usr/bin/contrail-api --conf_file=/etc/contrail/contrail-api.conf --disc_server_ip=127.0.0.1 1>>/var/log/contrail/contrail-api.stdout 2>>/var/log/contrail/contrail-api.err"],
```

Comment:
Sanju proposed that we should move contrail stuff to pods eventually.
Having every component talk to contrail-discovery, makes it easier to move
them around in future. It is also the most tested version in house, where
in we always use discovery in contrail setups during testing.

On Tue, Sep 1, 2015 at 3:32 PM, Pedro Marques <notifications@github.com>
wrote:

> In cluster/contrail-api.manifest
> <https://github.com/Juniper/contrail-kubernetes/pull/40#discussion_r38480390>
> :
>
> > @@ -7,7 +7,7 @@
> >  	"containers":[{
> >  	    "name": "contrail-api",
> >  	    "image": "opencontrail/config:2.20",
> > -	    "command": ["/bin/sh", "-c", "/usr/bin/contrail-api --conf_file=/etc/contrail/contrail-api.conf 1>>/var/log/contrail/contrail-api.stdout 2>>/var/log/contrail/contrail-api.err"],
> > +	    "command": ["/bin/sh", "-c", "/usr/bin/contrail-api --conf_file=/etc/contrail/contrail-api.conf --disc_server_ip=127.0.0.1 1>>/var/log/contrail/contrail-api.stdout 2>>/var/log/contrail/contrail-api.err"],
>
> There isn't an advantage to run discovery in this setups, correct ?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/Juniper/contrail-kubernetes/pull/40/files#r38480390>.
>